community_server/src/Template/Element/centToFloat.ctp

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+$cut_places = $precision - 2;
+$transformAmount = $cent;
+if($cut_places > 0) {
+    $transformAmount = floor($cent / pow(10, $cut_places));
+}
+echo $transformAmount / pow(10, $precision -  $cut_places);
```

Comment:
What does this do? `$cut_places` is 0 or negative. I do not understand what happens here.

Reply:
With GDD precision is 4 with GDT precision is 2.
Maybe I have choosen a wrong name here. 
For GDD cut_places is 2 and 134156 GDD cent will be in first step rounded down to
1341 and then to 13,41.
For GDT this extra step isn't needed.

Reply:
ok, just seems quite complicated - resolving this as well, ty ❤️

Reply:
I thought when I make it variable then I can make it direct as variable as possible, so it can be used easy also for similar variants.